lms/envs/common.py

```diff
@@ -51,11 +51,6 @@
 
 PLATFORM_FACEBOOK_ACCOUNT = "http://www.facebook.com/YourPlatformFacebookAccount"
 PLATFORM_TWITTER_ACCOUNT = "@YourPlatformTwitterAccount"
-PLATFORM_TWITTER_URL = "https://twitter.com/YourPlatformTwitterAccount"
```

Comment:
It looks like these are still used on the LMS course about page for white label.